src/renderer/components/ft-list-video/ft-list-video.vue

```diff
@@ -145,6 +145,28 @@
           class="viewCount"
         > • {{ $tc('Global.Counts.Watching Count', viewCount, {count: parsedViewCount}) }}</span>
       </div>
+      <div
+        v-if="is4k || hasCaptions"
+        class="videoTagLine"
+      >
+        <div
+          v-if="is4k"
+          class="videoTag"
+          :aria-label="$t('Search Listing.Label.4K')"
+          role="img"
+        >
+          {{ $tc('Search Listing.Label.4K') }}
+        </div>
+        <div
+          v-if="hasCaptions"
+          class="videoTag"
+          :aria-label="$t('Search Listing.Label.Closed Captions')"
+          role="img"
+          :style="{marginLeft: is4k ? '4px' : '0px' }"
```

Comment:
**suggestion (blocking):** Please use dynamic classes to apply styling adjustments (`:class={ /* */ }`). This allows better expansion of the affected styles, better semantic clarification of the intention/purpose, and also better linting due to some limitations of our current plugins. Case in point, `margin-left` instead of `margin-inline-start` not being enforced in the template, and thus leading to unexpected behavior for users using FT in an RTL language (e.g., Arabic).

Reply:
I've removed the conditional style scriptlet. I forgot about the CSS :first-child selector. The same thing is now implemented via CSS. 

The code exists to remove a weird indent on the video tags for the first element in the list.